src/nni_manager/rest_server/restHandler.ts

```diff
@@ -262,6 +263,16 @@ class NNIRestHandler {
         });
     }
 
+    private getLatestMetricData(router: Router): void {
+        router.get('/metric-data-latest/', async (req: Request, res: Response) => {
```

Comment:
In which file will you call this API? I does find the corresponding code.

Reply:
Updated.